worksheet_B/Mandelbrot.cpp

```diff
@@ -17,34 +17,72 @@ int main()
 	// Generate the image
 	for (int pixelY = 0; pixelY < image.height(); pixelY++)
 	{
-		// TODO: Map the y coordinate into the range minY to maxY
-		//double y0 =
+		
+		double y0 = (pixelY / 800.0) * (maxY - minY) + minY;
 
 		for (int pixelX = 0; pixelX < image.width(); pixelX++)
 		{
-			// TODO: Map the x coordinate into the range minX to maxX
-			//double x0 =
-
-			// TODO: implement the algorithm to colour a single pixel (x0, y0) of the Mandelbrot set fractal
-			// The code below simply fills the screen with random pixels
+		
+			double x0 = (pixelX / 800.0) * (maxX - minX) + minX;
 
+			double Xi = 0;
```

Comment:
Inconsistent naming: some of your variables start with lowercase letters, other with uppercase